### PR TITLE
Spotify for Doom Emacs

### PR DESCRIPTION
**Pull request by otavioschwanck**

### CODE REVIEW

File: modules/app/spotify/packages.el [hunk 1]
```diff
@@ -0,0 +1,17 @@
+;;; app/spotify/config.el -*- lexical-binding: t; -*-
+
+;;
+;;; Packages
+
+(package! helm-spotify-plus :pin "3383b1399a"
```
Comment (otavioschwanck) [inline modules/app/spotify/packages.el]: I dont now what is this pin, i put a random number.  Can anyone help me ?

File: modules/app/spotify/config.el [hunk 2]
```diff
@@ -0,0 +1,14 @@
+;;; app/spotify/config.el -*- lexical-binding: t; -*-
+
+(use-package! helm-spotify-plus
+  :when (featurep! helm)
+  :config
+  (map! :leader
+        :map global-map
+        :prefix "5"
```
Comment (otavioschwanck) [inline modules/app/spotify/config.el]: i accept suggestions on this prefix.

File: modules/app/spotify/packages.el (hunk 1 above)
Comment (yoavm448) [reply modules/app/spotify/packages.el]: Put your cursor inside those parentheses and execute `M-x doom/update-pinned-package-form`

Comment (otavioschwanck) [reply modules/app/spotify/packages.el]: Thanks!

**Comment by hlissner** [comment]
I cannot accept this PR because it's not enough to warrant a module. A user could trivially set it up themselves. It also only caters to helm users and does not accommodate ivy.

I don't mind a spotify module, but there has to be more to it.

Side notes: 
+ Modules should not define their own leader keybinds (leader keybinds should be bound in `:config default`)
+ `(featurep! helm)` should be `(featurep! :completion helm)`
+ The `helm-spotify-plus` isn't lazy loaded

**Pull request closed without merging**